### PR TITLE
Config for Heroku builds (Circle yet to come)

### LINKED ISSUE #3: Make a recommendation
As a user, when I click the button requesting a recommendation, I want to see a cocktail recommendation based on ratings I've provided and ratings from all other users in the system.

### PR DESCRIPTION
**Pull request by mkrumholz**
## Pull request checklist
Please check if your PR fulfills the following requirements:
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Rubocop has passed locally and any fixes were made for failures
- [ ] CircleCI build is passing (YAY!)
- [x] All tests passing locally

## Pull request type
Please check the type of change your PR introduces:
- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [x] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe):

## What is the current behavior?

Issue Number: 
closes #3 

App would not previously deploy on heroku, even manually with Python libraries available. 

## What is the new behavior?

- This should enable us to build the app manually for heroku. We need the PR merged in order to start merging in from main to heroku since it uses a script here to ensure that python is installed with shared libraries. We'll need to configure a procfile to deal with that for circleCI most likely, but this at least gets us to a live MVP. 

## Does this introduce a breaking change?
- [ ] Yes
- [x] No

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -1,15 +1,65 @@
...
-      - ruby/bundle-install
+  test:
+    executor: ruby/default
+    steps:
+      - checkout
+      - run: sudo apt-get install python3-pip
+      - run: python3 -m pip install numpy
+      - run: python3 -m pip install pandas
+      - run: python3 -m pip install sklearn
+      # - run:
```
Comment (markcyen) [inline .circleci/config.yml:37]: Interesting... so `run:` is like inputting the python3 stuff in the CLI?

File: app/services/recommendation_service.rb [hunk 2]
```diff
@@ -40,17 +39,9 @@ def self.euclidean_distance(df)
...
     user_distance_matrix = Pandas.DataFrame.new(data=euclidean, index=df.index,columns=df.index)
-    # binding.pry
   end
 
-  # def self.create_df
-  #   csv_data = Pandas.read_csv("./db/data/cocktail_ratings.csv") #replace with our ratings data
-  #   binding.pry
-  #   df = csv_data.set_index('name').fillna(0) #will be user_id instead of name
-  # end
-
```
Comment (markcyen) [inline app/services/recommendation_service.rb:51]: Great! Thanks for taking this out!

**Review by markcyen — APPROVED** [review]
Very cool. Thanks for digging into all this!

**Comment by twarbelow** [comment]
Do you still need this PR?

File: .circleci/config.yml (hunk 1 above)
Comment (mkrumholz) [reply .circleci/config.yml]: Yes! @markcyen it is basically running those commands on the server for circle ci

**Comment by mkrumholz** [comment]
> Do you still need this PR?

Just for the sake of recording this in GitHub as well, we do still need this PR, because we have to get this figured out in order to deploy. It is just not yet ready to be reviewed.

File: bin/post_compile [hunk 3]
```diff
@@ -0,0 +1,16 @@
+set -e
+buildpack_url=https://github.com/heroku/heroku-buildpack-python
+buildpack_vsn=v197
```
Comment (markcyen) [inline bin/post_compile:3]: Looks like the buildpack was needed like Dan suggested?

**Review by markcyen — APPROVED** [review]
Great job on this and can sense the struggles. Will pick your brain on process since this is way past my current understanding.

File: bin/post_compile (hunk 3 above)
Comment (mkrumholz) [reply bin/post_compile:3]: Yes! I also had to reorder everything and then this thing is what the woman on GitHub suggested, which is a script that, after Python builds, rebuilds it with shared libraries.

**Review by twarbelow — APPROVED** [review]
wow

**Pull request merged**